### PR TITLE
Milestone #3

### PR DESCRIPTION
**Pull request by jonathastavares**
In this milestone, we added simple styles for the Awesome Books page, also we created a script to make the list disappear when it is empty.

Built with:
- HTML
- CSS
- Bootstrap
- Javascript

### CODE REVIEW

File: assets/stylesheets/main.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  max-width: 35px;
+}
+
+.no-margin {
+  margin: 0;
+}
+
+.table > :not(caption) > * > * {
+  border: 0;
+}
```
Comment (DeliceLydia) [inline assets/stylesheets/main.css]: - [x] Please follow the given layouts check how the design looks like on this image
lay

your page
aw

As you can see layouts are differents please make necessary changes

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/stylesheets/main.css (hunk 1 above)
Comment (leolpaz) [reply assets/stylesheets/main.css]: Hey, thanks for the review, can we keep the table not displaying anything if it's empty?

Comment (DeliceLydia) [reply assets/stylesheets/main.css]: Hey @leolpaz,

Yes a table will display something after adding a book just follow the layouts only

**Comment by Davidosky007** [comment]
## Hi @jonathastavares @leolpaz 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**